.github/workflows/build-toolchain.yml

```diff
@@ -373,29 +373,33 @@ jobs:
                   "arch": "amd64",
                   "cpu": "x86_64",
                   "triple": "x86_64-unknown-windows-msvc",
-                  "compiler_target": "x86_64-unknown-windows-msvc",
                   "os": "Windows",
+                  "platform": "Windows",
                   "cc": "cl",
                   "cflags": "${{ steps.context.outputs.WINDOWS_CMAKE_C_FLAGS }}",
                   "cxx": "cl",
                   "cxxflags": "${{ steps.context.outputs.WINDOWS_CMAKE_CXX_FLAGS }}",
                   "swiftflags": "${{ steps.context.outputs.WINDOWS_CMAKE_Swift_FLAGS }}",
                   "cmake_linker_flags": "-D CMAKE_EXE_LINKER_FLAGS=\"${{ steps.context.outputs.WINDOWS_CMAKE_EXE_LINKER_FLAGS }}\" -D CMAKE_SHARED_LINKER_FLAGS=\"${{ steps.context.outputs.WINDOWS_CMAKE_SHARED_LINKER_FLAGS }}\"",
-                  "extra_flags": "-D CMAKE_SYSTEM_PROCESSOR=AMD64"
+                  "extra_flags": "-D CMAKE_SYSTEM_PROCESSOR=AMD64",
+                  "host_built_swiftc": "${HostToolchain}/usr/bin/swiftc.exe",
+                  "macros_swiftflags": "-strict-implicit-module-context -resource-dir ${SDKRoot}/usr/lib/swift -L${SDKRoot}/usr/lib/swift/windows -vfsoverlay ${WINDOWS_VFS_OVERLAY} -Xcc -Xclang -Xcc -fbuiltin-headers-in-system-modules"
```

Comment:
We should separately experiment with this and see if the `-strict-implicit-module-context` is needed at all.